nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -487,6 +541,10 @@ def forward(self, audio_signal, length, cache_last_channel=None, cache_last_time
                 _, pos_emb = self.pos_enc(x=audio_signal, cache_len=cache_len)
                 pad_mask, att_mask = self._create_masks(max_audio_length, length, audio_signal.device)
 
+            if lth in self.capture_output_at_layers and self.is_access_enabled():
```

Comment:
Can you add a comment with the registered tensor shapes as [B, T, D] or [B, D, T]?

Reply:
Plus, two things. One - the layer id part should be part of InterCTC Mixin's config management, not model specific. 
Two - after self.is_access_enabled() add a bool check to InterCTCMixin if the mixin is enabled or not. setup_interctc() should enable the flag depending on user's config setting.